main.tf

```
@@ -104,3 +104,20 @@ data "google_iam_policy" "policy" {
     }
   }
 }
+
+resource "google_project_iam_audit_config" "project" {
+  for_each = var.module_enabled ? var.audit_logs : []
```

better call variable like the block we use it for.

```suggestion
  for_each = var.module_enabled ? var.audit_configs : []
```